### PR TITLE
Create constrained-database-with-slow-streams.yml

### PR DESCRIPTION
**Pull request by ghost**
Let me know if the subject of my repo fits the up-for-grubs catalog

### CODE REVIEW

**Comment by ghost** [comment]
Someone please make this merge ? 
many thanks !!

File: _data/projects/constrained-database-with-slow-streams.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: Constrained Data Base with Slow Streams
+desc: Discussions (help-wanted) on usability of "Constrained Data Base with Slow Streams"
+site: https://github.com/bacloud14/Constrained-Data-Base-with-Slow-Streams
+tags:
+  - scientific
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/bacloud14/Constrained-Data-Base-with-Slow-Streams/labels/up-for-grubs
```
Comment (ritwik12) [inline _data/projects/constrained-database-with-slow-streams.yml]: @bacloud14 Hey, I think u misspelled the label name on your repo.
 ```suggestion
  link: https://github.com/bacloud14/Constrained-Data-Base-with-Slow-Streams/labels/up-for-grabs
```

File: _data/projects/constrained-database-with-slow-streams.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: Constrained Data Base with Slow Streams
+desc: Discussions (help-wanted) on usability of "Constrained Data Base with Slow Streams"
+site: https://github.com/bacloud14/Constrained-Data-Base-with-Slow-Streams
+tags:
+  - scientific
```
Comment (ritwik12) [inline _data/projects/constrained-database-with-slow-streams.yml:5]: @bacloud14 Can you please add more tags here. To make it more reachable.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/constrained-database-with-slow-streams.yml (hunk 2 above)
Comment (ghost) [reply _data/projects/constrained-database-with-slow-streams.yml:5]: thanks for pointing out

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@bacloud14 Thanks!!

**Pull request merged**